src/index.scss

```diff
@@ -0,0 +1,683 @@
+@import 'reset.scss';
+@import 'variables.scss';
+@import '_templates.scss';
+
+body {
+    @extend %base_style;
```

Comment:
Usually body - should not be touched at all. If you need to add styles to some global container, then you should add some div as 'global-wrapper', not touch body.